jmx-win-config.yml.sample

```diff
@@ -1,13 +1,12 @@
-integration_name: com.newrelic.jmx
-
-instances:
-  - name: jmx
-    command: all_data
-    arguments:
-      jmx_host: jmx-host.localnet
-      jmx_port: 9999
-      jmx_user: admin
-      jmx_pass: admin
-      collection_files: "C:\\Program Files\\New Relic\\newrelic-infra\\integrations.d\\jvm-metrics.yml"
-    labels:
-      env: staging
+integrations:
+- name: .\bin\nri-jmx.exe
+  env:
+    COLLECTION_FILES: "C:\Program Files\New Relic\newrelic-infra\integrations.d\jvm-metrics.yml"
```

Comment:
Not sure is this will break but we dropped 1 slash in the folder separators
i.e. `C:\\Program Files` to `C:\Program Files`

Reply:
definitely better to restore old notation, I'll check as well other integration to make sure we do not have the same issue

Reply:
I updated accordingly also NR_JMX